### PR TITLE
[New Rule] New GitHub App Installed

### PR DESCRIPTION
**Pull request by imays11**
## Issues

https://github.com/elastic/detection-rules/issues/3051
## Summary
This rule detects when a new GitHub App has been installed in your organization account. GitHub Apps extend GitHub's functionality both within and outside of GitHub. When an app is installed it is granted permissions to read or modify your repository and organization data. Only trusted apps should be installed and any newly installed apps should be investigated to verify their legitimacy. Unauthorized app installation could lower your organization's security posture and leave you exposed for future attacks.

## REVIEWERS
I've provided screenshots in the linked issue, however, to further support the review process I've created a mock GitHub organization that any reviewer can be added to. You can recreate the targeted behavior and all data is shipped to a shared stack. Please reach out to me if you'd like to take advantage of this option and I'll send an invite and provide the necessary credentials.

### CODE REVIEW

File: rules/integrations/github/execution_new_github_app_installed.toml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+from = "now-9m"
+index = ["logs-github.audit-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "New GitHub App Installed"
+risk_score = 47
+rule_id = "1ca62f14-4787-4913-b7af-df11745a49da"
+severity = "medium"
+tags = ["Domain: Cloud", "Use Case: Threat Detection", "Tactic: Execution", "Data Source: Github"]
+type = "eql"
```
Comment (Aegrah) [inline rules/integrations/github/execution_new_github_app_installed.toml:27]:
```suggestion
tags = ["Domain: Cloud", "Use Case: Threat Detection", "Tactic: Execution", "Data Source: Github"]
timestamp_override = "event.ingested"
type = "eql"
```

File: rules/integrations/github/execution_new_github_app_installed.toml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+query = '''
+configuration where event.module== "github" and event.action== "integration_installation.create"
```
Comment (Aegrah) [inline rules/integrations/github/execution_new_github_app_installed.toml:29]:
```suggestion
configuration where event.module == "github" and event.action == "integration_installation.create"
```

**Review by Aegrah — APPROVED** [review]
Nice, LGTM after unit testing is fixed.

**Review by terrancedejesus — APPROVED** [review]
Approving. @imays11 is there any specific key:value pair that suggests whether this is organization wide vs single user? Also, without the example data it is hard to tell but are there any details about the application installed that may indicate suspicious vs benign?

**Pull request merged**